dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -251,14 +257,18 @@ PageEntriesEdit BlobStore::write(DB::WriteBatch & wb, const WriteLimiterPtr & wr
     {
         auto blob_file = getBlobFile(blob_id);
         blob_file->write(buffer, offset_in_file, all_page_data_size, write_limiter);
+        fiu_do_on(FailPoints::exception_after_write_blob,
```

Comment:
This failpoint is seem like `force_set_page_file_write_errno`, Can we replace it?